parsers/US_NY.py

```diff
@@ -8,13 +8,20 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas',              # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
+                                                # <5% is oil burn from 2016 State of NYISO Market Report
            'Natural Gas': 'gas',
            'Nuclear': 'nuclear',
-           'Other Fossil Fuels': 'unknown',
-           'Other Renewables': 'unknown',
+           'Other Fossil Fuels': 'coal',    # NYISO states "Power produced by either coal or oil" - assume coal, oil only power plants only as last resort
+           'Other Renewables': 'biomass',   # NYISO states "Power produced by either solar energy, refuse, methane or wood"
```

Comment:
Following up on https://github.com/tmrowco/electricitymap/pull/942#discussion_r159057367:
@HansHyde unfortunately it's not simply a matter of trust as 3rd parties look at our data sources and want to verify for themselves. "Hans Hyde" as a source wouldn't work 😆 
Given the report you linked, it seems there is ~500MW of installed solar capacity at distribution level. That means that we can't exclude the fact that Other Renewables could contain an amount of solar significant compared to biomass.
I can seem to find solid evidence to definitely classify Other Renewables as biomass, so I'd keep it as unknown for now.

Reply:
Long response short - I'm pulling apart the 2017 NYISO "Gold Book" - It will answer our Generation capacities & fuel types questions no matter what (and whether we like the numbers or not) - data doesn't lie! :D  Spoiler alert - there are only 31.5 MW of installed PV reportable to NYISO, with very little in the pipeline.  "Counting" distributed (behind the meter) PV might be similar to "counting" Demand Response (like the 9,000 MW that PJM was reporting) - if there is no data, does it really exist? ;)

Reply:
I'm fine with that. We just need a solid reference to the 31.5 MW number you're reporting and add it the comment.